src/payment/strategies/klarna/klarna-payments.ts

```diff
@@ -0,0 +1,57 @@
+export default interface KlarnaPayments {
+    authorize(options: KlarnaAuthorizeOptions, data: KlarnaUpdateSessionParams, callback: (res: KlarnaAuthorizationResponse) => void): void;
```

Comment:
You can create only one data type instead of have two.

i.e.
`authorize(KlarnaOptions): KlarnaResponses`